README.md

```diff
@@ -30,7 +30,7 @@ that need installation.
     * pytest
     * pytest-cov
     * pillow
-    * pynput
```

Comment:
This may break our current implementation until Aitor pushes his changes to CLI using keyboard